portable/GCC/NiosII/port_asm.S

```diff
@@ -33,11 +33,10 @@
 .globl restore_sp_from_pxCurrentTCB		
 	
 # Entry point for exceptions.
-.section .exceptions.entry, "xa"		
+.section .exceptions.entry.user, "xa"
 
 # Save the entire context of a task.
 save_context:
-	addi	ea, ea, -4			# Point to the next instruction.

```

Comment:
Why was this removed?

Reply:
I remember that this line causes an error. (But I forgot why. I' m sorry.) 

To avoid this error,  the same expression written in BSP generated code was adopted.